src/frontend/src/components/public/PIASubHeader/index.tsx

```diff
@@ -133,10 +157,21 @@ function PIASubHeader({
       primaryButtonText === SubmitButtonTextEnum.DELEGATE_FINISH_REVIEW
```

Comment:
this logic is strange. Why can't we check if pia is delegated and the status is in complete review?

Reply:
i think both implementation are the same.